### DISCUSSION TITLE
IMSAI S-100 cards won't budge

### DISCUSSION THREAD

**Discussion by wfpunch** [thehighnibble/thehighnibble.github.io#88] [Q&A]
I just got my hands on an IMSAI PCS 80/30 and am trying to get started fixing it up. However, the cards have not been moved for 40 or so years and are really stuck. Of course, no clips on any card to help. I have applied a fair amount of physical force but the darn things won't budge. Any suggestions would be greatly appreciated.

**Comment by GrantMeStrength** [comment]
You're asking in the wrong place - this is a support group for a modern remake of a IMSAI which doesn't have S-100 cards.
I would suggest this google group: https://groups.google.com/g/s100computers

**Comment by wfpunch** [comment]
Thanks, that is very helpful.
